core/src/main/java/org/mule/runtime/core/internal/registry/MuleRegistryHelper.java

```diff
@@ -52,31 +53,31 @@
   /**
    * A reference to Mule's internal registry
    */
-  private Registry registry;
+  private final Registry registry;
 
   /**
    * We cache transformer searches so that we only search once
    */
   protected Map<String, Transformer> exactTransformerCache = new ConcurrentHashMap<>(8);
   protected Map<String, List<Transformer>> transformerListCache = new ConcurrentHashMap<>(8);
 
-  private MuleContext muleContext;
+  private final MuleContext muleContext;
```

Comment:
And this one

Reply:
it is being set in the constructor. The `final` is added automatically by the IDE when it detects that  it is not being written anywhere else.